### PR TITLE
Align naming of config options

### PR DESCRIPTION
**Pull request by dbenage-cx**
Changes most config options to group under common nodes.
This gives a more consistent layout of config.xml, improving readability and determination of config settings. (e.g. `UI.sound.enabled`->`UI.sound.effects.enabled`)

### CODE REVIEW

**Comment by dbenage-cx** [comment]
Sorted by old config value for easier comparison of changes.

Old | New
-----|-----
UI.auto-reposition-windows | UI.windows.auto-reposition
UI.ctrl-border-color | UI.controls.border-color
UI.ctrl-color | UI.controls.background-color
UI.default-link-color | UI.text.link.default-color
UI.detection-range-opacity | UI.map.detection-range.opacity
UI.dropdownlist-arrow-color | UI.controls.dropdownlist-arrow-color
UI.edit-hilite | UI.controls.edit-hilite-color
UI.field-fog-of-war-clr | UI.map.field.fog-of-war.color
UI.fleet-selection-indicator-size | UI.fleet.selection-indicator.size
UI.fleet-supply-line-dot-rate | UI.map.starlane.fleet-supply-line.dot-rate
UI.fleet-supply-line-dot-spacing | UI.map.starlane.fleet-supply-line.dot-spacing
UI.fleet-supply-line-width | UI.map.starlane.fleet-supply-line.width
UI.fleet-supply-lines | UI.map.starlane.fleet-supply-line.shown
UI.fleet-wnd-aggression | UI.fleet.initial-aggression
UI.fleet-wnd-scanline-clr | UI.fleet.scanline-solor
UI.font-bold | UI.font.bold
UI.font-size | UI.font.size
UI.galaxy-gas-background | UI.map.gas-background-shown
UI.galaxy-starfields | UI.map.starfields-shown
UI.hide-map-panels | UI.map.panels-shown (inverted)
UI.keypress-repeat-delay | UI.input.keyboard.key-repeat-delay
UI.keypress-repeat-interval | UI.input.keyboard.key-repeat-interval
UI.known-tech | UI.research.status.completed.color
UI.known-tech-border | UI.research.status.completed.border-color
UI.map-right-click-popup-menu | UI.map.right-click-menu-enabled
UI.medium-fleet-button-minimum-zoom | UI.map.zoom.fleet.medium-minimum
UI.mouse-click-repeat-delay | UI.input.mouse.click-repeat-delay
UI.mouse-click-repeat-interval | UI.input.mouse.click-repeat-interval
UI.multiple-fleet-windows | UI.fleet.use-multiple-windows
UI.objects-list-info-col-`#` | UI.objects-list.col-`#`
UI.objects-list-width-col-`#` | UI.objects-list.col-`#`.width
UI.queue-width | UI.queue.width
UI.research.listbox.`*` | UI.research.layout.list.`*`
UI.research.tech-tree-control.status.completed | UI.research.status.completed.shown
UI.research.tech-tree-control.status.has_researched_prereq | UI.research.status.has_prereq.shown
UI.research.tech-tree-control.status.researchable | UI.research.status.researchable.shown
UI.research.tech-tree-control.status.unresearchable | UI.research.status.unresearchable.shown
UI.researchable-tech | UI.research.status.researchable.color
UI.researchable-tech-border | UI.researchable.status.researchable.border-color
UI.resource-starlane-colouring | UI.map.starlane.use-empire-coloring
UI.rollover-link-color | UI.text.link.rollover-color
UI.scroll-width | UI.controls.scroll-width
UI.show-detection-range | UI.map.detection-range.shown
UI.show-fleet-eta | UI.fleet.eta-shown
UI.show-galaxy-map-scale | UI.map.scale-shown
UI.show-galaxy-map-scale-circle | UI.map.scale-circle-shown
UI.show-galaxy-map-zoom-slider | UI.map.zoom-slider-shown
UI.show-production-location-on-queue | UI.queue.production-location-shown
UI.show-unexplored_system_overlay | UI.map.system.unexplored.overlay-shown
UI.sidepanel-planet-fog-of-war-clr | UI.sidepanel.planet.fog-of-war-color
UI.sidepanel-planet-max-diameter | UI.sidepanel.planet.max-diameter
UI.sidepanel-planet-min-diameter | UI.sidepanel.planet.min-diameter
UI.sidepanel-planet-shown | UI.sidepanel.planet.shown
UI.sidepanel-width | UI.sidepanel.width
UI.sitrep-icon-size | UI.sitreps.icon-size
UI.small-fleet-button-minimum-zoom | UI.map.zoom.fleet.small-minimum
UI.sound.bg-music | UI.sound.music.background
UI.sound.music-enabled | UI.sound.music.enabled
UI.sound.music-volume | UI.sound.music.volume
UI.sound.new-turn.toggle | UI.sound.effects.new-turns.enabled
UI.sound.`*` (all others) | UI.sound.effects.`*`
UI.starlane-core-multiplier | UI.map.starlane.cor-multiplier
UI.starlane-thickness | UI.map.starlane.thickness
UI.stat-decrease-color | UI.controls.stat-decrease-color
UI.stat-increase-color | UI.controls.stat-increase-color
UI.state-button-color | UI.controls.state-button-color
UI.swap-mouse-lr | UI.input.mouse.swap-lr-buttons
UI.system-circle-distance | UI.map.system.circle.distance
UI.system-circle-size | UI.map.system.circle.size
UI.system-circles | UI.map.system.circle.shown
UI.system-fog-of-war | UI.map.system.fog-of-war.shown
UI.system-fog-of-war-clr | UI.map.system.fog-of-war.color
UI.system-fog-of-war-spacing | UI.map.system.fog-of-war.spacing
UI.system-icon-size | UI.map.system.icon-size
UI.system-inner-circle-max-width | UI.map.system.circle.inner.max-width
UI.system-inner-circle-width | UI.map.system.circle.inner.width
UI.system-name-unowned-color | UI.map.system.name,unowned-color
UI.system-outer-circle-width | UI.map.system.circle.outer.width
UI.system-selection-indicator-rpm | UI.map.zoom.system.selection-indicator.rpm
UI.system-selection-indicator-size | UI.map.zoom.system.selection-indicator.size
UI.system-tiny-icon-size-threshold | UI.map.zoom.system.tiny-icon-threshold
UI.tech-controls-graphic-size | UI.research.controls-graphics-size
UI.tech-layout-horz-spacing | UI.research.layout.tree.horz-spacing
UI.tech-layout-vert-spacing | UI.research.layout.tree.vert-spacing
UI.tech-layout-zoom-scale | UI.research.layout.tree.zoom-scale
UI.tech-progress | UI.research.has_progress.color
UI.tech-progress-background | UI.research.has_progress.background-color
UI.text-color | UI.text.color
UI.tiny-fleet-button-minimum-zoom | UI.map.zoom.fleet.tiny-minimum
UI.title-font | UI.font.title
UI.title-font-size | UI.font.title.size
UI.tooltip-delay | UI.tooltip.delay
UI.unowned-starlane-colour | UI.map.starlane.unowned-colour
UI.unresearchable-tech | UI.research.status.unresearchable.color
UI.unresearchable-tech-border | UI.research.status.unresearchable.border-color
UI.window-quickclose | UI.map.quickclose-windows
UI.wnd-color | UI.windows.color.background
UI.wnd-inner-border-color | UI.windows.color.inner-border
UI.wnd-outer-border-color | UI.windows.color.outer-border
app-height | client.fullscreen.height
app-width | client.fullscreen.width
app-left-windowed | client.windowed.left-edge
app-top-windowed | client.windowed.top-edge
app-width | client.fullscreen.width
app-width-windowed | client.windowed.width
auto-add-saved-designs | client.design.auto-add-saved
autosave.`*` | client.autosave.`*`
binary-serialization | save.serialization.binary-format
effect-accounting | client.effects.accounting
effects-threads-ai | client.effects.threads.ai
effects-threads-server | client.effects.threads.server
effects-threads-ui | client.effects.threads.ui
external-server-address | client.external-server.address
fake-mode-change | client.fake-mode-change
fullscreen-monitor-id | client.monitor-id
hidden-sitrep-templates | UI.sitreps.hidden-templates
limit-fps | fps.limit
limit-fps-no_focus | fps.no_focus.limit
max-fps | fps.limit.max
max-fps-no_focus | fps.no_focus.limit.max
reset-fullscreen-size | client.fullscreen.reset-size
show-fps | fps.shown
xml-zlib-serialization | save.serialization.xml-zlib-compression

**Comment by geoffthemedio** [comment]
There are some inconsistencies with verbs... in particular, are they imperative ("limit") or past-tense ("shown") ?

fps.limit.max seems redundant.

On the whole, it seems like you've made it more difficult to remember an option's name, due to mixing - and . (and sometimes _) as separators, instead of just using . to separate the relatively few categories of option.

autosaving is not client-specific.

client.effects.threads.server is self-contradictory.

**Comment by dbenage-cx** [comment]
Thanks geoff, looking at changing to the following in coming days:
* flags/booleans always labelled `shown` or `enabled`, depending on function.
* drop `client` parent for `autosave` (possibly group with save),  and for `effects`
* reduce some parent nodes where practical
* limiting the use of hyphens and underscores
* consistent `colour`/`color` name (later one unless anyone objects)

Any objection to CamelCase for multi-word names (`fog-of-war`) or prefer underscores?

**Comment by geoffthemedio** [comment]
keep all the options all lowercase

**Comment by adrianbroher** [comment]
I didn't check any of the details of this PS but the first thing that comes to mind for me:

Would the renaming allow to display settings by convention into the Options window (e.g: key end in `*.enable` would represented by checkboxes, `*.path` would display a file selection)?  Do the new names map to the existing Options placement within the settings menu (e.g: are `UI.*` settings located all in the corresponding `UI` tab within the options window)?

**Comment by dbenage-cx** [comment]
The names did not match to option tabs overall, I am reworking them to include both your points as well.

A potential oddity with the Colors tab: `UI.text-color` -> `colors.text.color`

(except for objects-list, each option will be in separate commit, so removing previous table)

**Comment by dbenage-cx** [comment]
working on fixups

File: default/stringtables/de.txt [hunk 1]
```diff
@@ -1115,7 +1115,7 @@ OPTIONS_DB_QUICKSTART
 Beginnt ein neues Schnellstartspiel. Das Hauptmenü wird umgangen.
 
 OPTIONS_DB_CHECKED_GL_VERSION
-Speichert ob die OpenGL Version des Systems überprüft wurde. Wenn nicht, werden einige Zeichenoptionen abhängig von der GL Version angepasst, nachdem die Version überprüft wurde.
+Speichert ob die OpenGL Version des Systems nicht überprüft wurde. Wenn falsche, werden einige Zeichenoptionen abhängig von der GL Version angepasst, nachdem die Version überprüft wurde.
```
Comment (dbenage-cx) [inline default/stringtables/de.txt]: Native speaker reviews requested on this commit for spanish, french, german, and russian.  The logic swapped but some of these do not seem to translate to show such logic to begin with.

If any lack review, I would opt to remove that particular translation than to have potentially incorrect description.

Comment (Vezzra) [reply default/stringtables/de.txt]: Please remove the German translation. The old version has already been incomplete (the entire part "...after which, this option will be set to true." is missing), and the rest isn't particularly good.

I don't have the time/am not in the mood to provide a reasonably well worded translation right now, let's leave that to the next volunteer who wants to take on the German translation. 😉

**Comment by dbenage-cx** [comment]
Old | New
-----|-----
`AI.config.trait.*.AI_*` | `ai.trait.*.ai_*`
`AI.config.trait.*.force` | `ai.trait.*.force_enabled`
`AI.config.trait.*.all` | `ai.trait.*.default`
version-string | version.string
`UI.windows.research.ResearchQueueWnd.*` | `ui.window.research.queue.*`
`UI.windows.production.ProductionQueueWnd.*` | `ui.window.production.queue.*`
`UI.windows.research.InfoPanel.*` | `ui.window.research.info_panel.*`
`UI.windows.production.InfoPanel.*` | `ui.window.production.info_panel.*`
`UI.windows.*.left` | `ui.window.*.fullscreen.left_edge`
`UI.windows.*.top` | `ui.window.*.fullscreen.top_edge`
`UI.windows.*.left-windowed` | `ui.window.*.windowed.left_edge`
`UI.windows.*.top-windowed` | `ui.window.*.windowed.top_edge`
`UI.windows.*.height` | `ui.window.*.fullscreen.height`
`UI.windows.*.width` | `ui.window.*.fullscreen.width`
`UI.windows.*.height-windowed` | `ui.window.*.windowed.height`
`UI.windows.*.width-windowed` | `ui.window.*.windowed.width`
`UI.windows.*.persistently-hidden` | `ui.window.*.persistently_hidden.enabled`
save-dir | dir.save.path
resource-dir | dir.resource.path
GameSetup.stars | ui.window.setup.star.count
GameSetup.galaxy-shape | ui.window.setup.galaxy.shape
GameSetup.galaxy-age | ui.window.setup.galaxy.age
GameSetup.planet-density | ui.window.setup.planet-density
GameSetup.starlane-frequency | ui.window.setup.starlane.frequency
GameSetup.specials-frequency | ui.window.setup.specials.frequency
GameSetup.monster-frequency | ui.window.setup.monster.frequency
GameSetup.native-frequency | ui.window.setup.native.frequency
GameSetup.empire-name | ui.window.setup.empire.name
GameSetup.player-name | ui.window.setup.player.name
GameSetup.empire-color | ui.window.setup.empire.color
GameSetup.starting-species | ui.window.setup.initial.species
GameSetup.ai-players | ui.window.setup.ai.player.count
GameSetup.ai-aggression | ui.window.setup.ai.aggression
GameSetup.seed | ui.window.setup.seed
hidden-strep-templates | ui.window.sitrep.hidden_templates.stringlist
UI.sitrep-icon-size | ui.window.sitrep.icon.size
UI.verbose-sitrep | ui.window.sitrep.invalid.shown
checked-gl-version | gl_version.check.enabled (inverted logic)
reseed-prng-server | server.prng.reseed.enabled
force-external-server | server.external.force.enabled
external-server-address | server.external.address
UI.save-file-dialog.* | ui.window.save_load.*
UI.research.listbox.column-widths.graphic | ui.window.research.list.column.graphic.width
UI.research.listbox.column-widths.name | ui.window.research.list.column.name.width
UI.research.listbox.column-widths.cost | ui.window.research.list.column.cost.width
UI.research.listbox.column-widths.time | ui.window.research.list.column.time.width
UI.research.listbox.column-widths.category | ui.window.research.list.column.category.width
UI.research.listbox.column-widths.description | ui.window.research.list.column.description.width
UI.tech-control-graphic-size | ui.window.research.control.graphic.size
UI.tech-layout-zoom-scale | ui.window.research.tree.zoom.scale
UI.tech-layout-vert-spacing | ui.window.research.tree.spacing.vertical
UI.tech-layout-horz-spacing | ui.window.research.tree.spacing.horizontal
UI.unresearchable-tech-border | research.unresearchable.border.color
UI.unresearchable-tech | research.unresearchable.background.color
UI.tech-progress | research.incomplete.progress.color
UI.tech-progress-background | research.incomplete.background.color
UI.researchable-tech | research.background.color
UI.researchable-tech-border | research.border.color
UI.known-tech-border | research.completed.border.color
UI.known-tech | research.completed.background.color
UI.research.tech-tree-control.status.researchable | research.shown
UI.research.tech-tree-control.status.has_researched_prereq | research.has_prereq.shown
UI.research.tech-tree-control.status.unresearchable | research.unresearchable.shown
UI.research.tech-tree-control.status.completed | research.completed.shown
UI.main-menu.x | ui.window.intro.menu.center.x
UI.main-menu.y | ui.window.intro.menu.center.y
UI.multiple-fleet-windows | ui.window.fleet.multiple.enabled
UI.fleet-wnd-scanline-clr | ui.window.fleet.scanline.color
UI.fleet-wnd-aggression | ui.window.fleet.aggression
UI.show-fleet-eta | ui.window.fleet.eta.shown
UI.encyclopedia.search.articles | ui.window.pedia.search.articles.enabled
UI.scroll-width | ui.control.scroll.width
UI.design-pedia-dynamic | ui.window.design.pedia.title.dynamic.enabled
UI.sidepanel-planet-shown | ui.window.sidepanel.planet.shown
UI.sidepanel-planet-max-diameter | ui.window.sidepanel.planet.diameter.max
UI.sidepanel-planet-min-diameter | ui.window.sidepanel.planet.diameter.min
UI.sidepanel-planet-fog-of-war-color | ui.window.sidepanel.planet.scanlane.color
UI.sidepanel-width | ui.window.sidepanel.width
effect-accounting | effects.accounting.enabled
auto-add-saved-designs | shipdesign.auto_add.saved.enabled
effects-threads-server | effects.server.threads
effects-threads-ai | effects.ai.threads
effects-threads-ui | effects.ui.threads
UI.hotkeys.* | ui.hotkeys.*
xml-zlib-serialization | save.serialization.compression.enabled
binary-serialization | save.serialization.binary.enabled
autosave.limit | save.auto.file.limit
autosave.turns | save.auto.interval.turns
autosave.multiplayer | save.auto.multiplayer.enabled
autosave.single_player | save.auto.single_player.enabled
UI.combat.* | ui.window.combat_report.*
UI.state-button-color | ui.control.button.state.color
UI.dropdownlist-arrow-color | ui.control.dropdownlist.arrow.color
UI.edit-hilite | ui.control.edit.highlight.color
UI.ctrl-border-color | ui.control.background.color
UI.ctrl-color | ui.control.background.color
UI.wnd-outer-border-color | ui.window.border.outer.color
UI.wnd-inner-border-color | ui.window.border.inner.color
UI.wnd-color | ui.window.background.color
UI.objects-list-info-col-* | ui.window.objects.columns.c*.stringkey
UI.objects-list-width-col-* | ui.window.objects.columns.c*.width
UI.hide-map-panels | ui.window.production.other_panels.removed
UI.show-unexplored_system_overlay | map.system.unexplored.rollover.enabled
UI.map-right-click-popup-menu | map.popup_menu.enabled
UI.detection-range-opacity | map.detection_range.opacity
UI.show-detection-range | map.detection_range.shown
UI.show-galaxy-map-zoom-slider | map.zoom.slider.shown
UI.show-galaxy-map-scale circle | map.scale.circle.shown
UI.show-galaxy-map-scale | map.scale.legend.shown
UI.galaxy-starfields | map.star_background.shown
UI.galaxy-gas-background | map.system.gas_background.shown
UI.unowned-starlane-colour | map.starlane.color
UI.starlane-core-multiplier | map.starlane.thickness.core_multiplier
UI.resource-starlane-colouring | map.starlane.empire.color.shown
UI.starlane-thickness | map.starlane.thickness
UI.fleet-supply-line-dot-rate | map.fleet.supply_line.dot.rate
UI.fleet-supply-line-dot-spacing | map.fleet.supply_line.dot.spacing
UI.fleet-supply-line-width | map.fleet.supply_line.width
UI.fleet-supply-lines | map.fleet.supply_line.shown
UI.fleet-selection-indicator-size | map.fleet.selection_indicator.size
UI.medium-fleet-button-minimum-zoom | map.fleet.button.medium.zoom_threshold
UI.small-fleet-button-minimum-zoom | map.fleet.button.small.zoom_threshold
UI.tiny-fleet-button-minimum-zoom | map.fleet.button.tiny.zoom_threshold
UI.field-fog-of-war-clr | map.field.scanlines.color
UI.system-fog-of-war-clr | map.system.scanlines.color
UI.system-fog-of-war-spacing | map.system.scanlines.spacing
UI.system-fog-of-war | map.scanlines.shown
UI.system-name-unowned-color | map.system.unowned.name.color
UI.system-tiny-icon-size-threshold | map.system.icon.size.tiny_threshold
UI.system-selection-indicator-rpm | map.system.selection_indicator.rpm
UI.system-selection-indicator-size | map.system.selection_indicator.size
UI.system-circle-distance | map.system.circle.distance
UI.system-inner-circle-max-width | map.system.circle.inner.width.max
UI.system-outer-circle-width | map.system.circle.outer.width
UI.system-inner-circle-width | map.system.circle.inner.width
UI.system-circle-size | map.system.circle.size
UI.system-circles | map.system.circle.shown
UI.system-icon-size | map.system.icon.size
UI.show-id-after-name | ui.name.id.shown
UI.dump-effects-descriptions | ui.description.effects.shown
UI.show-production-location-on-queue | ui.window.queue.production_location.shown
UI.queue-width | ui.window.queue.width
UI.stat-decrease-color | ui.font.stat.decrease.color
UI.stat-increase-color | ui.font.stat.increase.color
UI.rollover-link-color | ui.font.link.rollover.color
UI.default-link-color | ui.font.link.color
UI.text-color | ui.font.color
UI.font-bold | ui.font.bold.path
UI.title-font-size | ui.font.title.size
UI.title-font | ui.font.title.path
UI.font-size | ui.font.size
UI.font | ui.font.path
tooltip.extended-delay | ui.tooltip.extended_delay
tooltip-delay | ui.tooltip.delay
stringtable-filepname | ui.stringtable.path
UI.auto-reposition-windows | ui.window.reposition.enabled
UI.quickclose-windows | ui.window.quickclose.enabled
UI.mouse-click-repeat-interval | ui.input.mouse.button.repeat.interval
UI.mouse-click-repeat-delay | ui.input.mouse.button.repeat.delay
UI.keypres-repeat-interval | ui.input.keyboard.repeat.interval
UI.keypres-repeat-delay | ui.input.keyboard.repeat.delay
UI.swap-mouse-lr | ui.input.mouse.button_swap.enabled
UI.sound.new-turn.sound-file | audio.effects.new_turn.path
UI.sound.new-turn.toggle | audio.effects.new_turn.enabled
UI.sound.sidepanel-open | audio.effects.window.sidepanel.open.path
UI.sound.system-icon-rollover | audio.effects.icon.system.rollover.path
UI.sound.fleet-button-click | audio.effects.button.fleet.click.path
UI.sound.fleet-button-rollover | audio.effects.button.fleet.rollover.path
UI.sound.window-close | audio.effects.window.close.path
UI.sound.window-minimize | audio.effects.window.minimize.path
UI.sound.window-maximize | audio.effects.window.maximize.path
UI.sound.alert | audio.effects.alert.path
UI.sound.text-typing | audio.effects.keypress.path
UI.sound.list-pulldown | audio.effects.listbox.open.path
UI.sound.item-drop | audio.effects.listbox.drop.path
UI.sound.list-select | audio.effects.listbox.select.path
UI.sound.turn-button-click | audio.effects.button.turn.click.path
UI.sound.planet-button-click | (REMOVED)
UI.sound.button-click | audio.effects.button.click.path
UI.sound.button-rollover | audio.effects.button.rollover.path
UI.sound.bg-music | audio.music.path
UI.sound.music-volume | audio.music.volume
UI.sound.volume | audio.effects.volume
UI.sound.enabled | audio.effects.enabled
UI.sound.music-enabled | audio.music.enabled
reset-fullscreen-size | video.fullscreen.reset
limit-fps-no-focus | video.fps.unfocused.enabled
max-fps-no_focus | video.fps.unfocused
limit-fps | video.fps.max.enabled
max-fps | video.fps.max
show-fps | video.fps.shown
fullscreen-monitor-id | video.monitor.id
fake-mode-change | video.fake_mode.enabled
fullscreen | video.fullscreen.enabled
app-top-windowed | video.windowed.top_edge
app-left-windowed | video.windowed.left_edge
app-width-windowed | video.windowed.width
app-height-windowed | video.windowed.height
app-height | video.fullscreen.height
app-width | video.fullscreen.width

**Comment by adrianbroher** [comment]
Looks good so far, but what about the `ui.window.setup` subtree?  I would expect `setup.*` to be a separate coniguration root but it isn't.  Why do you associate it to `ui.window`?

**Comment by dbenage-cx** [comment]
Explicitly not touched:
`verbose-logging` and `verbose-combat-logging` to lower conflicts with logging PR (where they are removed/replaced).

All modified options are now lower case, with minimal use of underscores.

> Would the renaming allow to display settings by convention into the Options window (e.g: key end in *.enable would represented by checkboxes, *.path would display a file selection)?

Using end node conventions as follows:
boolen: `enabled`, `shown`, `reset` as appropriate
file location: `path`
strintagle lookup: `stringkey`
list of strings: `stringlist`

> Do the new names map to the existing Options placement within the settings menu (e.g: are UI.* settings located all in the corresponding UI tab within the options window)?

Started down this path, so most of the options on a page have that parent node(except colors), the reverse is not true.
Many entries are grouped instead to a common parent node, such as research status settings.
`ui` has many options with no GUI setting.  I've left the windows under this parent node, but happy to drop the parent if that is desired.

**Comment by dbenage-cx** [comment]
> ui.window.setup subtree?

I did so because the options revolved around that particular window, will change to `setup.` parent node.

**Comment by dbenage-cx** [comment]
Rebased for upstream changes with less formatting changes.

Considering moving audio effects under their respective control, replacing their root `audio.effects`.

**Comment by dbenage-cx** [comment]
Rebased for conflict resolution
[example config](https://gist.github.com/dbenage-cx/b4c0ad0e57ab42471e453c7d8b2d870b)

**Comment by Vezzra** [comment]
@dbenage-cx, status report on this?

**Comment by geoffthemedio** [comment]
I don't think this needs to be ~200 separate commits.

I suppose standardizing the names a bit is reasonable, but I'm not keep on having to remember different option names.

**Comment by dbenage-cx** [comment]
Suppose I could rewrite and group commits by the new top level node.

IMO trying to remember an option is already difficult. If we can find consensus on naming, an improved `--help` can be realized by breaking the options down into categories per node group.
e.g. `--help map.starlane` or possibly also `--help Starlane`

**Comment by geoffthemedio** [comment]
A shorter (single page) output from `-h` that lists a set of alternative help switches that given more specialized output, like `--help map` and `--help directories` or similar would be quite useful. The current huge multi-page spew of output from `-h` is awkward at best to find something in... And if actually wanted, `--help all` could be provided...

**Comment by Cjkjvfnby** [comment]
> A shorter (single page) output from -h that lists a set of alternative help switches 

I live in the world where `-h` and `--help` is same thing, making them different will be confusing for me.

May be make default behavior is to show group names and mention that `*` or `all` will show all groups, and use `-h group` to see its content.

**Comment by dbenage-cx** [comment]
Bad push, moved to separate branch.

Work on command-line output would be new PR, will get a forum thread open for it in coming days.

**Comment by Vezzra** [comment]
Superseded by #1831.

**Pull request closed without merging**